src/main/java/ch/njol/skript/expressions/ExprMaxDurability.java

```diff
@@ -18,46 +18,107 @@
  */
 package ch.njol.skript.expressions;
 
-import org.bukkit.inventory.ItemStack;
-import org.eclipse.jdt.annotation.Nullable;
-
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.bukkitutil.ItemUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.skript.util.slot.Slot;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Max Durability")
-@Description("The maximum durability of an item.")
-@Examples({"maximum durability of diamond sword",
-		   "if max durability of player's tool is not 0: # Item is damageable"})
-@Since("2.5")
-public class ExprMaxDurability extends SimplePropertyExpression<Object, Long> {
+@Description({"The maximum durability of an item. Changing requires Minecraft 1.20.5+",
+	"Note: 'delete' will remove the max durability from the item (making it a non-damageable item). Delete requires Paper 1.21+"})
+@Examples({
+	"maximum durability of diamond sword",
+	"if max durability of player's tool is not 0: # Item is damageable",
+	"set max durability of player's tool to 5000",
+	"add 5 to max durability of player's tool",
+	"reset max durability of player's tool",
+	"delete max durability of player's tool"
+})
+@Since("2.5, INSERT VERSION (change)")
```

Comment:
Needs @RequiredPlugins annotation with MC 1.20.5+

Reply:
oh right, im not used to doing that, haha, will do shortly